packages/cadl-rlc-emitter/src/transform/transformParameters.ts

```diff
@@ -154,8 +154,10 @@ function transformBodyParameters(
       {
         name: "body",
         type,
-        required: !Boolean(bodyParameters.optional),
-        description: getDoc(program, bodyParameters)
+        required: parameters.bodyParameter
+          ? !Boolean(parameters.bodyParameter.optional)
+          : bodySchema.required,
+        description: getDoc(program, bodyType)
```

Comment:
I don't think we should get description from bodyType because this way we'll get the model description not property description